doc/source/getting_started/index.rst

```diff
@@ -3,65 +3,65 @@
 Getting Started
 ===============
 
-Please, carefully read all the instructions in this page in order to achieve a
-successfully install of ``ansys-templates``.
+To successfully install ``pyansysansys-templates``, carefully read all instructions on this page.
 
 
 Before Installing
 -----------------
 
-Several requirements need to be fulfilled before you can proceed installing
-``ansys-templates`` tool.
+Several requirements must be met before you install
+``pyansys-templates``.
 
 
-Upgrading pip
+Upgrading Pip
```

Comment:
```suggestion
Upgrading pip
```

As the name of the tool, `pip` is always referred to in all-lowercase, as far as I'm aware. See e.g. their own documentation https://pip.pypa.io/en/stable/ The same might be true for `pipx`.

Reply:
OK--I'll tag it as a literal so it doesn't look like a missed initial capital letter.